pkg/daemon/daemon.go

```diff
@@ -495,13 +524,14 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 			nodeProfile:       *nodeProfile,
 			clockType:         clockType,
 			ptpClockThreshold: getPTPThreshold(nodeProfile),
+			haProfile:         haProfile,
```

Comment:
This is passed regardless of whether or not this is phc2sys?

Reply:
Yes ; it’s empty for all other use cases ;